src/services/user/services/ForcePasswordChange.js

```diff
@@ -0,0 +1,80 @@
+const { BadRequest } = require('@feathersjs/errors');
+const { disallow } = require('feathers-hooks-common');
+const { authenticate } = require('@feathersjs/authentication');
+const { passwordsMatch } = require('../../../utils/passwordHelpers');
+
+const ForcePasswordChangeServiceHooks = {
+	before: {
+		all: [],
+		find: disallow('external'),
+		get: disallow('external'),
+		create: [authenticate('jwt')],
+		update: disallow('external'),
+		patch: disallow('external'),
+		remove: disallow('external'),
+	},
+	after: {
+		all: [],
+		find: [],
+		get: [],
+		create: [],
+		update: [],
+		patch: [],
+		remove: [],
+	},
+};
+
+class ForcePasswordChangeService {
+	constructor(options) {
+		this.options = options || {};
+		this.err = Object.freeze({
+			missmatch: 'Die neuen Passwörter stimmen nicht überein.',
+			failed: 'Can not update the password. Please contact the administrator',
+		});
+	}
+
+	async newPasswordProvidedByUser(data, params) {
+		const newPassword = data['password-1'];
+		if (!passwordsMatch(newPassword, data['password-2'])) {
+			throw new BadRequest(this.err.missmatch);
+		}
+
+		const accountUpdate = {
+			password: newPassword,
+			userForcedToChangePassword: true,
+		};
+
+		const accountPromise = this.app.service('/accounts')
+			.patch(params.account._id, accountUpdate, params);
+
+		const userPromise = this.app.service('/users')
+			.patch(params.account.userId, { forcePasswordChange: false });
+
+		return Promise.allSettled([accountPromise, userPromise])
+			.then(([accountResponse, userResponse]) => {
+				if (accountResponse.status === 'rejected') {
+					throw new Error(accountResponse.reason);
+				}
+				if (userResponse.status === 'rejected') {
+					throw new Error(userResponse.reason);
+				}
+				return userResponse.value;
+			})
+			.catch((err) => {
+				throw new BadRequest(this.err.failed, err);
```

Comment:
if the pw change fails, but the user update doesnt, the forcePasswordChange is unset.
To avoid this, I would do the two requests sequencially, such that worst case is that the password is changed but the user continues to be forced to change the password. (in all other cases we end in a consistent state)

Reply:
:+1:  done